packages/httpsnippet-client-api/test/__datasets__/issue-76/har.json

```diff
@@ -3,25 +3,9 @@
     "entries": [
       {
         "request": {
-          "allHeaders": {
-            "accept": "application/xml",
-            "authorization": "Bearer 123"
```

Comment:
Removed this because the auth for this test is defined as a query string. `allHeaders` is also specific to [httpsnippet](http://npm.im/httpsnippet) so we weren't even using this in the snippet anyways.